src/main/java/core/basesyntax/service/impl/ReportConstructorImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportConstructor;
+import java.util.Map;
+
+public class ReportConstructorImpl implements ReportConstructor {
+    private static final String INTRODUCTION = "fruit, quantity";
+    private static final String CSV_SPLITTER = ",";
+
+    @Override
+    public String createReport(Map<Fruit, Integer> reportMap) {
+        StringBuilder stringBuilder = new StringBuilder();

```

Comment:
For the future, do not use such naming, how to guess what it stores?
```suggestion
        StringBuilder report = new StringBuilder();

```